locales/en/index.yml

```
@@ -2978,6 +2978,10 @@ features:
       status:
         failure: FALLITA
         success: ""
+  itWallet:
+    discovery:
+      title: Start with your Digital Identity
+      content: "Use SPID or CIE credentials to add your Digital Identity to the Wallet. **This is a necessary step to add your digital documents** (e.g. driving license) securely.\n### Good to know\nIn the future, you will be able to use the Digital Identity saved in your Wallet to identify yourself through IT Wallet functionality. \n\nFor more information, read the [Privacy Policy]() and the [Terms and Conditions of Use]()"
```

```suggestion
  itWallet:
    discovery:
      info:
        title: Start with your Digital Identity
        content: "Use SPID or CIE credentials to add your Digital Identity to the Wallet. **This is a necessary step to add your digital documents** (e.g. driving license) securely.\n### Good to know\nIn the future, you will be able to use the Digital Identity saved in your Wallet to identify yourself through IT Wallet functionality. \n\nFor more information, read the [Privacy Policy]() and the [Terms and Conditions of Use]()"
```
This will allow to add more keys for other screens in the `discovery` flow. What do you think?

---

Solved [here](https://github.com/pagopa/io-app/pull/5696/commits/9abd9c5fa423294ca87f32940834424ea7351660)